app/lib/json_web_token.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class JsonWebToken
+  SECRET_KEY = Rails.application.secrets.secret_key_base. to_s
+
+  def self.encode(payload, exp = 24.hours.from_now)
```

Comment:
Use public key encrption here so that we can verify the token at client level, example https://github.com/tachyons/authentication-service/blob/master/app/services/json_web_token.rb